ports/voidstranger/gameinfo.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Void Stranger.sh</path>
+    <name>Void Stranger</name>
+    <desc>Void Stranger is a 2D sokoban-style puzzle game where every step counts.
+
+Descend into the forgotten labyrinth teeming with fiendish foes and traps that defy reason. A swift defeat lurks at every corner for those who neither study their surroundings nor think their moves through with care.
+Trust you wits and slowly, surely, you will conquer the mysteries before you.
+
+You do remember why you are here, right?
+Or have you lost something?
+Something very important to you.</desc>
+    <releasedate>20230901T000000</releasedate>
+    <developer>System Erasure</developer>
+    <publisher>System Erasure</publisher>
+    <genre>Puzzle</genre>
+    <image>./cover.png</image>
```

Comment:
Needs path with cover

Reply:
changed back, not sure what I was thinking